lib/chef-acceptance/application.rb

```diff
@@ -52,8 +57,8 @@ def run(suite_regex, command)
         output_formatter.add_row(suite: "Run", command: "Total", duration: total_duration, error: run_error?)
         print_summary
       rescue AcceptanceFatalError => e
-        puts ""
-        puts e
+        log ""
```

Comment:
just a thought for stdout readability only.  `logger.stdout_logger.info ""`  I'm being OCD here.